src/utils/DocumentElementHelper.js

```diff
@@ -0,0 +1,24 @@
+import HTMLElement from '../HTMLElement';
+
+/**
+ * Class provide processing for innerHTML function for Document.documentElement
+ * element. When this element property innerHTML changes, it also updates links
+ * for head, body elements in document.
+ */
+export default class DocumentElementHelper extends HTMLElement {
```

Comment:
why do you call it Helper, place it in utils, and it is an Element ?
Perhaps HtmlElement is a better name and placed under the HTMLElement folder ?